### PR TITLE
Task 6 - SQS & SNS, async microservices communication

### PR DESCRIPTION
**Pull request by machulan**
[Task 6](https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task6-sqs-sns/task.md)

CloudFront:
https://d15zr8xzdj76yx.cloudfront.net

All subtasks except unit tests are done.

### CODE REVIEW

File: product-service/serverless.ts [hunk 1]
```diff
@@ -68,8 +93,71 @@ const serverlessConfiguration: Serverless = {
...
+      SNSSubscription: {
+        Type: 'AWS::SNS::Subscription',
+        Properties: {
+          Endpoint: 'nik.machula.aws@gmail.com',
+          Protocol: 'email',
+          TopicArn: {
+            Ref: 'SNSTopic',
+          },
+        },
+      },
```
Comment (sleepwalky) [inline product-service/serverless.ts]: Are you going to specify filter policy?

**Review by sleepwalky — APPROVED** [review]
No tests, no filter policy

Correct?

File: product-service/serverless.ts (hunk 1 above)
Comment (machulan) [reply product-service/serverless.ts]: I have just added the notification filter policy. Could you review?

Comment (sleepwalky) [reply product-service/serverless.ts]: 👍

**Pull request merged**